internal/cli/flagset/flagset.go

```diff
@@ -217,8 +218,8 @@ func (i *SliceStringFlag) String() string {
 }
 
 func (i *SliceStringFlag) Set(value string) error {
-	*i.Value = append(*i.Value, strings.Split(value, ",")...)
-
+	// overwritting insted of appending
+	*i.Value = strings.Split(value, ",")
```

Comment:
So we are expecting that a user will provide a particular flag only once. In that case, we overwrite the default values, instead of appending the user-passed values to the default ones.

Reply:
Also, the default values will be used if flags are not provided.